dm/worker/worker_test.go

```diff
@@ -85,6 +85,12 @@ func (t *testServer) TestTaskAutoResume(c *C) {
 		taskName = "sub-task-name"
 		port     = 8263
 	)
+	hostName := "127.0.0.1:8291"
```

Comment:
How about using `tempurl.Alloc()`?

Reply:
This url is from the config file `dm-worker.toml`.